README.md

```diff
@@ -1,65 +1,36 @@
-# Conduit Connector Template
+# Conduit Connector SFTP
 
-This is a template project for building [Conduit](https://conduit.io) connectors in Go. It makes it possible to
-start working on a Conduit connector in a matter of seconds.
+The SFTP connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It
+provides both, a source and a destination SFTP connector.
 
-## Quick start
+## How to build it
 
-1. Click [_Use this template_](https://github.com/new?template_name=conduit-connector-template&template_owner=ConduitIO) and clone your new repository.
-2. Initialize the repository using [`setup.sh`](https://github.com/ConduitIO/conduit-connector-template/blob/main/setup.sh) and commit your changes.
-   ```sh
-   ./setup.sh github.com/myusername/conduit-connector-myconnector
-   git add -A
-   git commit -m "initialize repository"
-   ```
-3. Set up [automatic Dependabot PR merges](#automatically-merging-dependabot-prs).
+Run `make build`.
 
-With that, you're all set up and ready to start working on your connector! As a next step, we recommend that you 
-check out the [Conduit Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk).
+## Testing
 
-## What's included?
+Run `make test` to run all the unit and integration tests.
 
-* Skeleton code for the connector's configuration, source and destination.
-* Example unit tests.
-* A [Makefile](/Makefile) with commonly used targets.
-* A [GitHub workflow](/.github/workflows/test.yml) to build the code and run the tests.
-* A [GitHub workflow](/.github/workflows/lint.yml) to run a pre-configured set of linters.
-* A [GitHub workflow](/.github/workflows/release.yml) which automatically creates a release when a tag is pushed.
-* A [Dependabot setup](/.github/dependabot.yml) which checks your dependencies for available updates and 
-[merges minor version upgrades](/.github/workflows/dependabot-auto-merge-go.yml) automatically.
-* [Issue](/.github/ISSUE_TEMPLATE) and [PR templates](/.github/pull_request_template.md).
-* A [README template](/README_TEMPLATE.md).
+## Source
 
-## Automatically merging Dependabot PRs
+The source SFTP connector monitors a directory on an SFTP server for files matching a specified pattern. It reads these files and converts them into `opencdc.Record` that can be processed by Conduit. The connector supports both password and private key authentication methods.
```

Comment:
Handling large files (through chunking) is a good feature, and I think it's worth mentioning here.